LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadData.cs

```diff
@@ -326,15 +328,33 @@ public void ChangeEndianess()
         /// </summary>
         /// <remarks>
         /// src https://github.com/jieter/python-lora/blob/master/lora/crypto.py.</remarks>
-        public byte[] GetDecryptedPayload(string appSkey)
```

Comment:
the name suggests we use the appSKey? How come we change this to use the NetworkSessionKey?

Reply:
sorry, I see that we have 2 overloads. all good.